setup.cfg

```diff
@@ -8,3 +8,7 @@ import-order-style = smarkets
 [coverage:run]
 source = user_deletion
 omit = *tests*
+
+[coverage:report]
+show_missing = True
+fail_under = 100
```

Comment:
seems like `fail_under` fails on my local machine, it sounds like a bug, I'm going to remove it for now.